index.js

```diff
@@ -251,15 +240,23 @@ function BigAssFanAccessory(log, config) {
     return (value ? Characteristic.RotationDirection.COUNTER_CLOCKWISE : Characteristic.RotationDirection.CLOCKWISE);
   }
 
+  var occupancySetWrapper = function(value) {
```

Comment:
Also don't be sorry about it :P I just wasn't sure if _I_ was missing something :)